developer_manual/app/advanced/code_signing.rst

```diff
@@ -31,8 +31,8 @@ upgrades will not prevent ownCloud from running, but will display a warning on
 the Admin page. For applications that are not tagged "Official" the code signing 
 process is optional.
 
-Not Open Source Anymore?
-^^^^^^^^^^^^^^^^^^^^^^^^
+Is ownCloud Not Open Source Anymore?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The ownCloud project is open source and always will be. The code signing 
 process is optional, though highly recommended. The code check for the 
```

Comment:
Thanks for letting me know. This section was just a minor rework of what was already there. I'll make the required change.